modules/ding_nodelist/README.md

```diff
@@ -0,0 +1,12 @@
+Ding nodelist
+===========
+
+Nodelist module for Alpha project. Allows to build customizeable lists
```

Comment:
"Alpha project" has no meaning within ding2. Please remove it from the documentation to avoid confusion.